### PR TITLE
SLES-15-010030 rule

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add rule for SLES-15-010030

#### Rationale:

- Map package_vsftpd_removed rule to STIG ID SLES-15-010030

### CODE REVIEW

File: linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml [hunk 1]
```diff
@@ -29,6 +30,10 @@ references:
     iso27001-2013: A.12.1.2,A.12.5.1,A.12.6.2,A.14.2.2,A.14.2.3,A.14.2.4,A.9.1.2
     cis-csc: 11,14,3,9
     stigid@rhel8: RHEL-08-040360
+    nist@sle15: IA-5(1)(c),IA-5(1).1(v),CM-7,CM-7.1(ii),CM-7 a
```
Comment (brett060102) [inline linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml:33]: CM-7 a should be CM-7(a)

**Review by brett060102 — CHANGES_REQUESTED** [review]
small change, see inline comment.

File: linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml (hunk 1 above)
Comment (teacup-on-rockingchair) [reply linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml]: Thanks  - should be ok in f9febde30d3516c821d74f9054e855f7ea5c185c

**Review by brett060102 — COMMENTED** [review]
LGTM

**Comment by ggbecker** [comment]
@openscap-ci ok to test

**Review by ggbecker — APPROVED** [review]

**Pull request merged**